cvm/quantization/tfm_ops.py

```diff
@@ -1189,10 +1190,10 @@ def quantize(self, op, **kwargs):
         # scales[out.attr('name')] = scales[X_name]
         # return out
 
-        precs[name][OUT_KEY] = precs[X.attr('name')][OUT_KEY]
         scales[name] = iscale = scales[X.attr('name')]
         a_min = sutils.get_attr(attrs, "a_min") * iscale
         a_max = sutils.get_attr(attrs, "a_max") * iscale
+        precs[name][OUT_KEY] = min(get_bit(th_dict[name]*iscale), get_bit(int(a_max)))
```

Comment:
How the tight precision influence the quantized model? Is it necessary, just for confusion?

Reply:
The original precs for `clip` is not the tight one. After restoring all the `convolution` and `clip`, the accuracy looks good. Howver, while using `clip` quantization and keeping `convolution` restored, the accuracy was poor.
When clip was switched to tight precision, the accuracy works well for all the lite models.

Reply:
mobilenetv2_1.0 works well no matter `clip` is restored or not, which is strange.

Reply:
mobilenetv2_1.0 works well no matter prec out_key for `clip` is tight or not, which is strange.

Reply:
And clip only exist in mobilenetv2_1.0 and lite models

Reply:
That's cause op clip is converted from ReLu6, mobilenetv2 use this operator in its network, and many lite models seem to prefer the ReLu6 op, don't know and just guess it. As you point out, the clip op that is necessary for tight or not relates to the requant&restore pass for post-quantization accuracy promotion, is it? I have created a new card about the post-quantization accuracy promotion in the [Trello](https://trello.com/c/gxsqqFZ6/3-accuracy-finetune-via-layers-restore) you'd better check out.